### ISSUE TITLE
Braces of case sections are aligned to case keyword

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#520]
Braces of case sections in switch statements should be aligned to the case keyword.

For example, following should produce a warning:
```c#
case 42:
  {
  }
```

while following should NOT produce a warning:
```c#
case 42:
{
}
```
